orttraining/orttraining/test/graph/gradient_graph_builder_test.cc

```diff
@@ -1087,14 +1087,28 @@ void RetrieveSendRecvOperators(
   for (auto& node : graph.Nodes()) {
     if (node.OpType() == "Send") {
       if (is_backward(node)) {
+        // backward_send can only be assigned one valid pointer.
+        // If it is assigned more than once, it means we have multiple
+        // Send in backward pass and therefore our assumption doesn't hold. 
+        // This check ensure that only we only update *backward_send when
```

Comment:
>only  [](start = 34, length = 5)

remove?

Reply:
Ah.. Sure. Thanks.

---
In reply to: [426743594](https://github.com/microsoft/onnxruntime/pull/3964#discussion_r426743594) [](ancestors = 426743594)